resource/specs/testnet.toml

```
@@ -33,7 +33,7 @@ code_hash = "0xb35557e7e9854206f7bc13e3c3a7fa4cf8892c84a09237fb0aab40aab3771eee"
 args = []
 
 [params]
-epoch_reward = 5_000_000_000_000_000
+epoch_reward = 500_000_000_000_000
```

It's hard to read a number in unit Shannon like this, I suggest group the last 8 digits like this:

```suggestion
epoch_reward = 5_000_000_00000000
```

It should also be applied to `secondary_epoch_reward`